eng/pipelines/common/templates/runtimes/build-runtime-tests-and-send-to-helix.yml

```diff
@@ -38,39 +38,48 @@ steps:
       archType: ${{ parameters.archType }}
       buildConfig: ${{ parameters.buildConfig }}
       testBuildArgs: ${{ parameters.testBuildArgs }}
-
   # Build a Mono LLVM AOT cross-compiler for non-amd64 targets (in this case, just arm64)
```

Comment:
Nit. since we are using this for mini  full AOT now, we can remove the "LLVM" from the comment